kayak_core/src/color.rs

```diff
@@ -19,30 +19,27 @@ impl Default for Color {
 
 impl Color {
     /// The black color.
-    pub const BLACK: Color = Color {
-        r: 0.0,
-        g: 0.0,
-        b: 0.0,
-        a: 1.0,
-    };
+    pub const BLACK: Color = Color::rgb(0.0, 0.0, 0.0);
```

Comment:
We can also do something like:
```rust
#[cfg(not(feature = "bevy_renderer"))]
pub use color::Color;
#[cfg(feature = "bevy_renderer")]
pub use bevy::prelude::Color;
```

Reply:
This would force us to ensure we have a similar API to bevy anyways , though, right? Otherwise in all the other crates and for the default widgets, we'll have to add these feature lines as well (not terrible but not super ideal). 

I’d prefer adding `From<Color>` impls for both under the bevy feature. Then the user can choose to use either at any time and we don't have to feature-flag internal color-related code.

Reply:
I like the `From<Color>` idea, but I'm not familiar enough with rust to do that. I'm not exactly sure how that would look like. Would I just need to impl the From trait on color::Color and that's it?

Reply:
No problem! Yep, something like:

```rust
#[cfg(feature = "bevy_renderer")]
impl From<bevy::prelude::Color> for crate::Color { /* ... */ }
```

And then the same for the reverse.

Reply:
Lets go with `From<Color>` :)